docs/tools/.readthedocs.yaml

```
@@ -9,8 +9,18 @@ version: 2
 build:
   os: ubuntu-22.04
   tools:
-    python: "3.11"
+    python: "3.12"
   jobs:
+    post_checkout:
+      # Cancel building pull requests when there aren't changed in the docs directory or YAML file.
+      #
+      # If there are no changes (git diff exits with 0) we force the command to return with 183.
+      # This is a special exit code on Read the Docs that will cancel the build immediately.
```

The reasoning behind the exit code is super nerdy. Maybe a link to this doc?

```suggestion
      # https://docs.readthedocs.io/en/latest/build-customization.html#cancel-build-based-on-a-condition
      # If there are no changes (git diff exits with 0) we force the command to return with 183.
      # This is a special exit code on Read the Docs that will cancel the build immediately.
```

---

```python
>>> sum(list("skip".encode("ascii")))
439
>>> 439 % 256
183
```

---

It really **is** super nerdy, but Okay